openpower/configs/witherspoon_defconfig

```diff
@@ -72,3 +72,4 @@ BR2_OCC_GPU_BIN_BUILD=y
 BR2_CAPP_UCODE_BIN_FILENAME="cappucode.bin"
 BR2_PACKAGE_LOADKEYS=y
 BR2_IMA_CATALOG_DTS="POWER9"
+BR2_OPENPOWER_SIGNED_SECURITY_VERSION="0"
```

Comment:
So we don't want this to be applied upstream first?

Reply:
No, we don't want this applied upstream first.